content/en/user-guide/authentication/index.md

```diff
@@ -0,0 +1,71 @@
+---
+title: Authentication
+linkTitle: Authentication
+weight: 22
+description: Get started with authentication in Snowflake.
+---
+
+## Overview
+
+Snowflake supports [multiple authentication methods](https://docs.snowflake.com/en/user-guide/authentication-policies). The Snowflake emulator supports the following authentication methods:
+
+* Username and password
+* RSA key pair authentication
+
+This guide demonstrates how to use the Snowflake emulator to authenticate using both methods.
+
+## Username and password
+
+To authenticate using a username and password, you can set the `user` and `password` parameters in the connection string. The values for these parameters can be set to `test` in the Snowflake emulator. Since the Snowflake emulator is a local instance, the username and password can be the same, and the authentication mechanism is mocked.
```

Comment:
worth adding that a default user and password could be changed using `SF_DEFAULT_USER` and `SF_DEFAULT_PASSWORD` https://localstack-snowflake-docs-preview-pr-79.surge.sh/references/configuration/#core